tasks/BamToGvcf.wdl

```diff
@@ -0,0 +1,109 @@
+version 1.0
+
+# Local Import
+#import "GermlineVarianltDiscovery.wdl" as Calling
+#import "Qc.wdl" as QC
+#import "Utilities.wdl" as Utils
+
+# Git URL Import
+import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.1.0/tasks/GermlineVarianltDiscovery.wdl" as Calling
+import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.1.0/tasks/Qc.wdl" as QC
+import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.1.0/tasks/Utilities.wdl" as Utils
+
+workflow BamToGvcf {
+
+  input {
+    File calling_interval_list
+    File evaluation_interval_list
+    Int haplotype_scatter_count
+    Int break_bands_at_multiples_of
+    Float? contamination
+    File input_bam
+    File ref_fasta
+    File ref_fasta_index
+    File ref_dict
+    File dbsnp_vcf
+    File dbsnp_vcf_index
+    String base_file_name
+    String final_gvcf_base_name
+    Int agg_preemptible_tries
+  }
+
+  # Break the calling interval_list into sub-intervals
+  # Perform variant calling on the sub-intervals, and then gather the results
+  call Utils.ScatterIntervalList as ScatterIntervalList {
+    input:
+      interval_list = calling_interval_list,
+      scatter_count = haplotype_scatter_count,
+      break_bands_at_multiples_of = break_bands_at_multiples_of
+  }
+
+  # We need disk to localize the sharded input and output due to the scatter for HaplotypeCaller.
+  # If we take the number we are scattering by and reduce by 20 we will have enough disk space
+  # to account for the fact that the data is quite uneven across the shards.
+  Int potential_hc_divisor = ScatterIntervalList.interval_count - 20
+  Int hc_divisor = if potential_hc_divisor > 1 then potential_hc_divisor else 1
+
+  # Call variants in parallel over WGS calling intervals
+  scatter (index in range(ScatterIntervalList.interval_count)) {
+    # Generate GVCF by interval
+    call Calling.HaplotypeCaller_GATK4_VCF as HaplotypeCaller {
```

Comment:
We're not running this version yet in production.  Does anyone care?  Maybe not.